src/main/java/new_metrics/CustomBetaDistribution.java

```diff
@@ -25,20 +10,18 @@
 import org.apache.commons.math3.util.FastMath;
 import org.apache.commons.math3.util.Precision;
 
-import new_metrics.*;
-
 /**
- * Implements the Beta distribution.
+ * Implements the Beta distribution. Customized from org.apache.commons.commons-math3 BetaDistribution.
  *
  * @see <a href="http://en.wikipedia.org/wiki/Beta_distribution">Beta distribution</a>
  * @since 2.0 (changed to concrete class in 3.0)
  */
-public class BetaDistribution extends AbstractRealDistribution {
```

Comment:
I think we could eventually compose a `CustomBetaDistribution` object with a `BetaDistribution` object so as to delegate as much logic from the former to the latter and not have to maintain a separate copy. But that can be done in a future PR.